src/routes/api/oba/shape/[shapeId]/+server.js

```diff
@@ -1,24 +1,6 @@
-import { error, json } from '@sveltejs/kit';
-import { PUBLIC_OBA_SERVER_URL as baseURL } from '$env/static/public';
-import { PRIVATE_OBA_API_KEY as apiKey } from '$env/static/private';
+import shape from '../../../../../lib/RestAPI/shape.js';
 
 export async function GET({ params }) {
-	const { shapeId } = params;
-
-	let apiURL = `${baseURL}/api/where/shape/${shapeId}.json?key=${apiKey}`;
-
-	try {
-		const response = await fetch(apiURL);
-		console.log('response:', response);
-
-		if (!response.ok) {
-			throw new Error(`HTTP error! status: ${response.status}`);
-		}
-
-		const data = await response.json();
-		return json(data);
-	} catch (err) {
-		console.error('Error fetching shape data:', err);
-		throw error(500, 'Unable to fetch shape data.');
-	}
+	const shapeId = params.shapeId;
```

Comment:
I'm not certain that the abstraction of having `$lib/RestAPI/shape.js` buys us anything here in terms of decreased complexity. I think I'd rather see that logic expressed here without the intermediary step. And I think the same should probably go for the other files under `RestAPI` unless they are DRYing up a lot of extra logic.